camera:

  config_camera: [4306.37205655706, 0.0, 802.1519317540232, 
                     0.0, 4307.411849731741, 479.2531644403244, 
                     0.0, 0.0, 1.0]
  camera_width: 1920
  camera_height: 1200

